Mathlib/Data/Matroid/Basic.lean

```diff
@@ -229,6 +229,9 @@ theorem ground_nonempty (M : Matroid α) [M.Nonempty] : M.E.Nonempty :=
 theorem ground_nonempty_iff (M : Matroid α) : M.E.Nonempty ↔ M.Nonempty :=
   ⟨fun h ↦ ⟨h⟩, fun ⟨h⟩ ↦ h⟩
 
+lemma nonempty_type (M : Matroid α) [h : M.Nonempty] : Nonempty α :=
```

Comment:
Hmm I'm not quite convinced of this name - I don't think we often use `_type` in this way. But I don't have any better suggestions, so I'll leave it up to you.